src/modules/Dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -33,7 +44,42 @@ export default class Dashboard extends Component {
                         </div>
                     </div>
                 </section>
+                <div className="container-fluid pl-3 pr-3">
+                    <div className="row">
+                        {Object.keys(imagesList).map((imageId) => (
+                            <div className="col-xl-6" key={imageId}>
+                                <CardImage
+                                    key={imageId}
+                                    userImage={imagesList[imageId]}
+                                    imageId={imageId}
+                                />
+                            </div>
+                        ))}
+                    </div>
+                </div>
+                <NotesModal />
             </Regular>
         );
     }
 }
+
+Dashboard.propTypes = {
+    imagesList: PropTypes.objectOf(PropTypes.object),
+    fetchUserImagesList: PropTypes.func.isRequired,
+};
+
+Dashboard.defaultProps = {
+    imagesList: {},
+};
+
+const mapStateToProps = ({ dashboard }) => ({
+    imagesList: dashboard.imagesList,
+});
+
+const mapDispatchToProps = (dispatch) => ({
+    fetchUserImagesList: () => {
```

Comment:
You can write it as `fetchUserImagesList: () => dispatch(fetchUserImagesListAPI()),`.